cpp/src/nvgraph/include/Louvain_matching.cuh

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <rmm/rmm.h>
+#include <rmm/thrust_rmm_allocator.h>
+
```

Comment:
No comments or doc in this file? I think that would help a lot with understanding and maintenance.
A few lines on how this new code addresses the issues in the former version (`nvgraph/include/delta_modularity.cuh` i guess) would help as well.